### PR TITLE
Fix route location link

### PR DESCRIPTION
**Pull request by fxiang1**
Signed-off-by: Feng Xiang <fxiang@redhat.com>

Issue: https://github.com/stolostron/backlog/issues/23531

- Fix route link so it displays again in the node sidebar

### CODE REVIEW

**Comment by fxiang1** [comment]
Will hold off on the merge until Nelson approves.

**Review by chenz4027 — APPROVED** [review]
lgtm, btw @fxiang1 I guess you would cherry pick it in 2.6 as well?

**Pull request merged**
